HierarchicalDataConverter/Catalyst.HierarchicalDataConverter.AutomatedTests/GrandparentParentChildTest.cs

```diff
@@ -0,0 +1,441 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Catalyst.HierarchicalDataConverter.AutomatedTests
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    using Catalyst.DataProcessing.Shared.Models.Enums;
+    using Catalyst.DataProcessing.Shared.Models.Metadata;
+    using Catalyst.DataProcessing.Shared.Utilities.Client;
+
+    using DataConverter;
+
+    using Fabric.Databus.Config;
+
+    using Moq;
+
+    [TestClass]
+    public class GrandparentParentChildTest
+    {
+        [TestMethod]
+        public void TestMethod1()
```

Comment:
better name for the test?

Reply:
That's weird - I thought I had renamed it.  I'll make sure it's checked in.